README.md

```diff
@@ -42,10 +43,16 @@ source thirdparty/set_thirdparty_env.sh
 ```
 
 The provided script `setup_build_env.sh` sets up a build environment for you
-with third party dependencies.  You use it by running `source
-setup_build_env.sh`.  By default, it will create a build directory `build/`.
-You can override the build directory by setting the BUILD_DIR env variable to
-another location.
+with third party dependencies.  You use it by running
+
+```shell
+source setup_build_env.sh
+```
+
+(You can't build Thrift on macOS with this script! Use Homebrew.)
```

Comment:
Maybe this merits 2 sections (, 1 for Linux and 1 for OS X) with a single set of shell commands to copy and paste for each? May slightly improve the bounce rate on following the instructions